src/views/topics/GdpTopic17View.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+    <TopicMeta :topic="topic" />
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+import { indexTopicMap } from '@/data';
+import { Topic } from '@/types';
+
+const topic = indexTopicMap.get(17) as Topic;
+
+export default defineComponent(
+    {
+        setup() {
+            return { topic };
+        },
+        mounted() {
```

Comment:
nit: if mounted is not needed for now, just remove it

Reply:
Sure! Actually, I am thinking that is it possible to make the topic routes dynamically so we don't need to add every route one by one?

Reply:
Yes and no. By "dynamically", I think you mean searching for "Views" automatically by name in the corresponding directory and bind the component to a route.

The issue is we do can render a list of routes according to the pre-defined topic list, but we have to pass in the "...View" file as the component to be rendered on the page.

Since we have to register component manually anyways; maybe make it explicit? It is not gonna take a lot of effort. Or maybe you have better ideas? We can discuss and see what might be better.

Reply:
I think you're right. Just adding new routes is more explicit. It will make the program become more complicated if using dynamic route.